frameworks/PHP/php-ngx/deploy/nginx_async.conf

```diff
@@ -0,0 +1,68 @@
+user www-data;
+worker_processes  auto;
+error_log stderr error;
+#worker_rlimit_nofile 100000;
+#timer_resolution 1000ms;
+daemon off;
+
+events {
+    worker_connections 65536;
+	multi_accept off;	 
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    access_log off;
+    server_tokens off;
+
+    sendfile off;
+    tcp_nopush off;
+    tcp_nodelay on;
+    keepalive_timeout 65s;
+    keepalive_disable none;
+    keepalive_requests 10000;
+
+    php_ini_path /deploy/conf/php.ini;
+
+    init_worker_by_php '
+        define("DB_HOST", gethostbyname("tfb-database"));
+        define("DB_PORT", "3306");
+        define("DB_USER", "benchmarkdbuser");
+        define("DB_PASS", "benchmarkdbpass");
+        define("DB_NAME", "hello_world");
+        
+        require "/ngx_php7/t/lib/mysql.php";
```

Comment:
I will move that code in `app-async.php`, to clean the nginx config. 
It's the app file that need to require and config the dependencies.

Reply:
> I would move that code in `app-async.php`, to clean the nginx config.
> It's the app file that need to require and config the dependencies.

@joanhey Fixed it by https://github.com/TechEmpower/FrameworkBenchmarks/pull/5138/commits/8458069eff8f1efd237ad5f0934e221a659e0147